Server/MusiciansAPP/MusiciansAPP.sln

```diff
@@ -0,0 +1,43 @@
+﻿
```

Comment:
Let's move the whole MusiciansAPP folder to the root folder.
It's a more common way.